lib/cdo/script_constants.rb

```diff
@@ -51,14 +60,48 @@ def ScriptConstants.hoc?(name)
         MINECRAFT_NAME,
         INFINITY_NAME,
         ARTIST_NAME
-    ].include? name
+    ],
+    csf: [
+        TWENTY_HOUR_NAME,
+        COURSE1_NAME,
+        COURSE2_NAME,
+        COURSE3_NAME,
+        COURSE4_NAME
+    ],
+    csp: [
+        CSP_UNIT1_NAME,
+        CSP_UNIT2_NAME,
+        CSP_UNIT3_NAME,
+        CSP_UNIT4_NAME,
+        CSP_UNIT5_NAME,
+        CSP_UNIT6_NAME
+    ],
+    math: [
+      ALGEBRA_NAME,
+      ALGEBRA_A_NAME,
+      ALGEBRA_B_NAME
+    ],
+    twenty_hour: [TWENTY_HOUR_NAME],
+    flappy: [FLAPPY_NAME],
+    minecraft: [MINECRAFT_NAME]
+  }
+
+  def self.script_in_category?(category, script)
+    return CATEGORIES[category].include? script
```

Comment:
I'm happy with that behavior. I expect this will continue to be used mostly with hardcoded categories, so passing in a nonexistant category is almost certainly a bug. Better to blow up than just silently return `false`.